src/models/Task.ts

```diff
@@ -1,11 +1,13 @@
 import { Timestamp } from 'firebase/firestore';
+import { DocumentReference } from 'firebase/firestore';
 
 export interface Task {
   name: string;
   description: string;
   importance: number;
   deadline: Timestamp;
-  active: boolean;
+  archived: boolean;
   status: boolean;
   uuid: string;
+  ref: DocumentReference;
```

Comment:
Hmmm.. I wonder what's the reason for this property

Reply:
that one is for updating task's info in the database